static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.tsx

```diff
@@ -57,14 +67,11 @@ function WidgetBuilderV2({
 
   const isSmallScreen = useMedia(`(max-width: ${theme.breakpoints.small})`);
 
-  const [{translate}, setTranslate] = useState<{
-    initialTranslate: Translate;
-    translate: Translate;
-  }>({
-    initialTranslate: {x: 0, y: 0},
-    translate: {x: 0, y: 0},
-  });
+  const [translate, setTranslate] = useState<WidgetDragPositioning>(
+    DEFAULT_WIDGET_DRAG_POSITIONING
+  );
 
+  // do we want to keep this?
```

Comment:
Good question, I don't know but let's just keep it around for now. It'll be easy to remove. I think one thing we should consider is raising a popup if there are unsaved changes

Reply:
ooo yes! Gonna make a sub-ticket for it so we don't forget.